include/inviwo/core/util/sourcecontext.h

```diff
@@ -95,16 +95,16 @@ std::basic_ostream<Elem, Traits>& operator<<(std::basic_ostream<Elem, Traits>& s
     return ss;
 }
 
-#define IVW_CONTEXT                                                                         \
-    SourceContext(parseTypeIdName(std::string(typeid(this).name())), std::string(__FILE__), \
+#define IVW_CONTEXT                                                                               \
+    SourceContext(util::parseTypeIdName(std::string(typeid(this).name())), std::string(__FILE__), \
                   std::string(__FUNCTION__), __LINE__)
 
 #define IVW_CONTEXT_CUSTOM(source) \
     SourceContext(source, std::string(__FILE__), std::string(__FUNCTION__), __LINE__)
 
 // Old deprecated macro, use uppercase
-#define IvwContext                                                                          \
-    SourceContext(parseTypeIdName(std::string(typeid(this).name())), std::string(__FILE__), \
+#define IvwContext                                                                                \
+    SourceContext(util::parseTypeIdName(std::string(typeid(this).name())), std::string(__FILE__), \
```

Comment:
parseTypeIdName should take a const char* as argument since that is what typeid().name() returns...